lib/platform/azure/index.ts

```diff
@@ -487,7 +488,7 @@ export async function mergePr(pr: number) {
 
 export function getPrBody(input: string) {
   // Remove any HTML we use
-  return input
+  return smartTruncate(input, 4000)
```

Comment:
4000 seems low compared to the others

Reply:
got it from here
https://github.com/renovatebot/renovate/blob/ee305b1a1719981684980399535789dbfe0494a2/lib/platform/azure/azure-helper.ts#L189-L194

Reply:
it checks out

Reply:
https://github.com/renovatebot/renovate/blob/8554df5c612ba5035c4f9b7061a4b45ca59b3dbb/lib/platform/azure/index.ts#L357

Reply:
https://developercommunityapi.westus.cloudapp.azure.com/content/problem/608770/remove-4000-character-limit-on-pull-request-descri.html